app/src/main/java/com/bogdan/codeforceswatcher/activity/MainActivity.kt

```diff
@@ -34,6 +36,8 @@ class MainActivity : AppCompatActivity() {
             startAlarm()
         }
 
+        UserLoader.loadUsers(CwApp.app.userDao.getAll(), false) {}
```

Comment:
I think, you can just omit `{}` here if default value for callback is provided in `loadUsers`.